recOrder/scripts/samples.py

```diff
@@ -1,29 +1,51 @@
 import shutil
 from pathlib import Path
 
+from typing import Literal
 from iohub import open_ome_zarr
+from iohub.ngff import Plate
 from napari.utils.notifications import show_warning
 from platformdirs import user_data_dir
 from wget import download
 
 
-def download_and_unzip() -> tuple[Path]:
+def _build_layer_list(dataset: Plate, layer_names: list[str]):
+    layer_list = []
```

Comment:
Is the actual expected return type [`napari.types.LayerDataTuple`](https://napari.org/stable/plugins/guides.html#the-layerdata-tuple) instead of a list?

Reply:
Oops I missed a level... this is a `list[LayerDataTuple'`.